packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -84,6 +84,8 @@ export class GrpcPlugin extends BasePlugin {
   static readonly ATTRIBUTE_GRPC_STATUS_CODE = 'grpc.status_code';
   static readonly ATTRIBUTE_GRPC_ERROR_NAME = 'grpc.error_name';
   static readonly ATTRIBUTE_GRPC_ERROR_MESSAGE = 'grpc.error_message';
+  private sentSeqId = 1;
```

Comment:
My reading of the specs is that these counters are supposed to be per-request so that the messages for a given request can be correlated between services. Would it be possible to handle these on a per request basis?

Reply:
Ah! You're right! I checked in other libraries. In [Go](https://github.com/census-instrumentation/opencensus-go/blob/264a2a48d94c062252389fffbc308ba555e35166/plugin/ocgrpc/trace_common.go#L93), id is 0 for all requests. In [Python](https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-grpc/opencensus/ext/grpc/utils.py#L31), wraps  a request or response iterator to add message events to the span. Looks like Java has most accurate implementation, But I am not sure how it translate previous hop count (seq) on wire. I will talk to @songy23 and @rghetia to understand it better.